supercursor-framework/src/personas/devops-engineer-persona.ts

```
@@ -0,0 +1,607 @@
+
+      // トリガータイプによる評価
+      let triggerScore = 0;
+      switch (context.trigger.type) {
+        case 'command':
+          const command = context.command?.toLowerCase() || '';
+          if (command.includes('deploy') || command.includes('ci') || command.includes('cd') ||
+              command.includes('docker') || command.includes('k8s') || command.includes('kubernetes') ||
+              command.includes('infrastructure') || command.includes('devops')) {
+            triggerScore += 35;
+          }
+          break;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)